README.md

```diff
@@ -11,6 +11,33 @@ You will require the SDK on your classpath, so go ahead and add it to your gradl
 implementation("io.getunleash:unleash-android-proxy-sdk:${unleash.sdk.version}")
 ```
 
+### Add support for minSdk < 24
+
+```kotlin
+android {
+    defaultConfig {
+        // Required when setting minSdkVersion to 20 or lower
+        multiDexEnabled = true
+    }
+
+    compileOptions {
+        // Flag to enable support for the new language APIs
+        coreLibraryDesugaringEnabled = true
+        // Sets Java compatibility to Java 8
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+}
+dependencies {
+    // ... 
+    coreLibraryDesugaring("com.android.tools:desugar_jdk_libs:1.1.5")
+    
+    // ... 
+}
+```
+Reference: https://developer.android.com/studio/write/java8-support
+
```

Comment:
I am unsure if all app developers would want to configure their apps with backward compatibility? 

If we only need `java.time.Duration` and just a subset of that implementation, could we instead provide a small custom implementation of that class, just in order to support `seconds` and `milliseconds` without ambiguity.

What do you think @afurtak?

Reply:
I can work on this, when I find some free time (possibly this week), but feel free to takeover the change if issue related to it is urgent. 

`java.times.Duration` is part of the library interface. It's not necessary to change it, but it shouldn't be used internally. Under the hood it's converted to just an integer, so I think, we can store the duration just as an integer without any wrapper. 

`java.time.Duration` is not only problem here. I've added one dependency in order to replace CompletableFuture implementation. 

Another thing was `java.nio.Files` usage, for which I provided alternative implementation of `Files.createTempDirectory("unleash_toggles").toFile()`

Also, maybe it's good idea to add AndromManifest file, with defined minSdk? Now I haven't seen any information about supported sdk versions in the docs (but i might have overlooked something). I can do it in this PR

Reply:
Thanks so much for this, we're going to merge it, but the README change will not be needed, we'll remove the dependency on duration. We're going to be aiming for the same compatibility level as OkHTTP (21), which should remove the need for the desugaring